### PR TITLE
homework27

### PR DESCRIPTION
**Pull request by OleksiiFltv**

### CODE REVIEW

File: tests/mockProfile.spec.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    await route.fulfill({
+      contentType: "application/json",
+      body: JSON.stringify(modifiedResponse),
+    });
+  });
+
+  await profileBtn.click();
+
+  const profileName = await page.locator(".profile_name.display-4");
+  const textContent = await profileName.textContent();
```
Comment (sttaran) [inline tests/mockProfile.spec.js]: remove it

File: tests/mockProfile.spec.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  expect(textContent).toContain("Andrew MockedUser");
```
Comment (sttaran) [inline tests/mockProfile.spec.js]: use https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text

File: tests/mockProfile.spec.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    await route.fulfill({
+      contentType: "application/json",
+      body: JSON.stringify(modifiedResponse),
+    });
+  });
+
+  await profileBtn.click();
+
+  const profileName = await page.locator(".profile_name.display-4");
```
Comment (sttaran) [inline tests/mockProfile.spec.js]: .profile_name would be enough

File: tests/mockProfile.spec.js (hunk 2 above)
Comment (sttaran) [inline tests/mockProfile.spec.js]: reuse data from `modifiedResponse`

**Review by sttaran — CHANGES_REQUESTED** [review]

**Comment by OleksiiFltv** [comment]
fixed. thanks for your help

**Review by sttaran — APPROVED** [review]

**Pull request merged**